src/ci-line-chart.js

```diff
@@ -256,16 +263,17 @@ const vis = {
     Array.from(pivotFieldNames).forEach((group, idx) => {
       // Filter data to the current group
       const groupData = d3data.filter((d) => d.pivot === group);
+      const groupLabel = group != 'NONE' ? ` - ${group}` : '';
```

Comment:
```suggestion
      const groupLabel = group !== 'NONE' ? ` - ${group}` : '';
```
Prefer strict equality check over abstract equality.